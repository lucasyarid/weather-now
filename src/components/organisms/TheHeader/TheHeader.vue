<template>
  <div class="TheHeader">
    <div class="TheHeader__logo">
      <BaseLogo />
    </div>
  </div>
</template>

<script>
import { BaseLogo } from "@/components/atoms";

export default {
  name: "TheHeader",
  components: {
    BaseLogo
  }
};
</script>

<style lang="scss" scoped>
.TheHeader {
  position: fixed;
  width: 100%;
  height: 50px;
  z-index: 100;
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: white;
  box-shadow: 0px 0px 5px 0px $color-box-shadow;
  &__logo {
    width: 150px;
  }
}
</style>
