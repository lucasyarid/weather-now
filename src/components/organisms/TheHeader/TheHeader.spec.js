import { shallowMount } from "@vue/test-utils";
import TheHeader from "./TheHeader";

describe("TheHeader", () => {
  const WRAPPER = shallowMount(TheHeader);

  test("Componente existe", () => {
    expect(WRAPPER.exists()).toBe(true);
  });

  test("Componente nÃ£o alterado", () => {
    expect(WRAPPER.html()).toMatchSnapshot();
  });
});
