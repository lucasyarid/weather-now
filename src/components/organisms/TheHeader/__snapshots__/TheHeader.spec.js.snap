// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeader Componente n√£o alterado 1`] = `"<div class=\\"TheHeader\\"><div class=\\"TheHeader__logo\\"><baselogo-stub variation=\\"default\\"></baselogo-stub></div></div>"`;
